pkg/webhook/cvc.go

```diff
@@ -130,6 +131,23 @@ func validatePoolListChanges(cvcOldObj, cvcNewObj *apis.CStorVolumeClaim) error
 			)
 		}
 	}
+	// Reject the request if someone perform scaling when CVC is not in Bound
+	// state
+	// NOTE: We should not reject the controller request which Updates status as
+	// Bound as well as pool info in status and spec
+	// TODO: Make below check as cvcOldObj.ISBound()
+	// If CVC Status is not bound then reject
+	if cvcOldObj.Status.Phase != apis.CStorVolumeClaimPhaseBound {
+		// If controller is updating pool info then new CVC will be in bound state
+		if cvcNewObj.Status.Phase != apis.CStorVolumeClaimPhaseBound &&
```

Comment:
you can remove this check

Reply:
We can't remove this check if we remove then update from CVC controller filling [pool info](https://github.com/openebs/maya/blob/3c21a6b2d0779c80fc43767cdc3dd15cd7ae07e2/cmd/cvc-operator/controller/cvc_controller.go#L322) will fail.
https://github.com/openebs/maya/blob/b9c0771a65071c54090f278f90839ba10cc1f8e9/pkg/webhook/cvc_test.go#L107